scopesim/effects/metis_lms_trace_list.py

```diff
@@ -307,8 +315,7 @@ def fov_grid(self):
         y_min = aperture["bottom"]
         y_max = aperture["top"]
 
-        filename_det_layout = from_currsys("!DET.layout.file_name", cmds=self.cmds)
```

Comment:
Ah here you introduced the bug that you then fixed in https://github.com/AstarVienna/irdb/pull/165 . We can merge that one if we also merge this one. Let me review the rest of this PR first though